sesman/startwm.sh

```diff
@@ -80,15 +84,23 @@ wm_start()
   # debian
   if [ -r /etc/X11/Xsession ]; then
     pre_start
-    
+
+    # if /etc/xrdp/export_desktop_session file exists, souurce it.
+    # /etc/xrdp/export_desktop_session script exports DESKTOP_SESSION environment varible
+    # The value shall be one of "ls -1 /usr/share/xsessions|cut -d. -f1".
+    # e.g. export DESKTOP_SESSION=ubuntu
+    if [ -r /etc/xrdp/export_desktop_session ]; then
+      . /etc/xrdp/export_desktop_session
+    fi
+
```

Comment:
As discussed in the PR, please remove this bit.

By all means add a comment to point the user to `~/.profile` or `/etc/profile.d/`